.github/workflows/project.yml

```diff
@@ -0,0 +1,23 @@
+name: Update projects
+on:
+  schedule:
+    - cron: '0 3,9,15,21 * * *'
```

Comment:
Should this only be once a day rather than four times a day?

Reply:
No, I intentionally made it more frequent. We want to be able to use the board during the day, not only during the status meetings. So we want it to be kept updated as much as possible. Maybe in the future, we will add per issue or per PR actions. Then we might make those less frequent. But so far running an action on each PR feels more resource-consuming than this than this.